app/assets/stylesheets/layout.scss

```diff
@@ -329,3 +330,11 @@ td, .table-list li { font-size: 14px; }
 #homepage-content {
   padding: 20px;
 }
+
+/* Search */
```

Comment:
Is this the left sidebar where you input your search, or the right box that shows the results? ("Search input", "Search results")